ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
What happens depends on this code: 

    {% if pulp_smash_version %}
    "version": "{{ pulp_smash_version }}",
    {% endif %}

If I remember my time with jinja correctly, references to non-existent variables are interpreted as being `None`. If that's the case, then no `"version": "…",` is placed in the config file. And that's fine.

> The `verify` and `version` keys are completely optional. … By default, Pulp Smash assumes that the Pulp server under test is theabsolute latest development version, but the `version` option can be used to explicitly run tests suitable for an older version of Pulp.